src/test/java/api/loans/InHouseUseCheckInTest.java

```diff
@@ -0,0 +1,169 @@
+package api.loans;
+
+import static api.support.http.CqlQuery.queryFromTemplate;
+import static api.support.http.Limit.noLimit;
+import static api.support.http.Offset.noOffset;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+
+import java.util.UUID;
+
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.CheckInByBarcodeResponse;
+import api.support.MultipleJsonRecords;
+import api.support.builders.RequestBuilder;
+
+public class InHouseUseCheckInTest extends APITests {
+
+  @Test
+  public void setInHouseUseTrue() {
```

Comment:
Please can this be reworded to describe under what circumstances the check in is considered in-house use.